dlint/util.py

```diff
@@ -1,21 +1,11 @@
 #!/usr/bin/env python
 
 import abc
-import sys
 
-if sys.version_info >= (3, 4):
-    ABC = abc.ABC
-else:
-    ABC = abc.ABCMeta(str('ABC'), (), {})
+ABC = abc.ABC
```

Comment:
Good catch :+1: although let's update the code to simply use `abc.ABC` everywhere and delete `util.ABC` now that it's just an alias for that class.